observatory-dags/observatory/dags/database/schema/jstor_institution_2021-03-01.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "description": "Institution name.",
+    "mode": "NULLABLE",
+    "name": "Institution",
+    "type": "STRING"
+  },
+  {
+    "description": "Title of the book.",
+    "mode": "REQUIRED",
+    "name": "Book_Title",
+    "type": "STRING"
+  },
+  {
+    "description": "DOI of the book on JSTOR.",
+    "mode": "NULLABLE",
+    "name": "Book_ID",
+    "type": "STRING"
+  },
+  {
+    "description": "Author of the book.",
+    "mode": "NULLABLE",
+    "name": "Authors",
+    "type": "STRING"
+  },
+  {
+    "description": "ISBN of the book (13 digits).",
+    "mode": "NULLABLE",
+    "name": "ISBN",
```

Comment:
Would be helpful if this was type STRING

Reply:
Thanks for picking that up, I'll check the other mellon telescopes as well to be sure.